dev_tools/autogenerate-bloqs-notebooks.py

```diff
@@ -129,25 +129,16 @@
         ],
         directory=f'{SOURCE_DIR}/bloqs/factoring',
     ),
-    NotebookSpec(
-        title='Tensor Hypercontraction',
-        module=qualtran.bloqs.chemistry.thc,
-        gate_specs=[
-            BloqNbSpec(qualtran.bloqs.chemistry.thc.prepare_test._make_uniform_superposition),
-            BloqNbSpec(qualtran.bloqs.chemistry.thc.prepare_test._make_prepare),
-            BloqNbSpec(qualtran.bloqs.chemistry.thc.select_test._make_select),
-        ],
-        directory=f'{SOURCE_DIR}/bloqs/chemistry/thc',
-    ),
-    NotebookSpec(
-        title='Sparse Hamiltonian',
-        module=qualtran.bloqs.chemistry.sparse,
-        gate_specs=[
-            BloqNbSpec(qualtran.bloqs.chemistry.sparse.prepare_test._make_sparse_prepare),
-            BloqNbSpec(qualtran.bloqs.chemistry.sparse.select_test._make_sparse_select),
-        ],
-        directory=f'{SOURCE_DIR}/bloqs/chemistry/sparse',
-    ),
+    # NotebookSpec(
```

Comment:
is this supposed to be deleted?

Reply:
The deleted stuff is supposed to be deleted, as it has been migrated to the `-v2` script.

@fdmalone should chime in: but it looks like the commented out part has been _partially_ migrated to the `-v2` script. The commented-out code does not run; it contains references to deleted example-making functions

Reply:
you should probably just delete the commented out code too?

Reply:
oh it hasn't been migrated, let me fix that.